src/components/layout/tables/AntTable/AntTable.tsx

```diff
@@ -0,0 +1,172 @@
+import { useEffect, useState } from 'react';
+
+import { useIsMobile } from '../../../../hooks';
+import { defaultDataProvider } from '../../../../services/arweave';
+import { useGlobalState } from '../../../../state/contexts/GlobalState';
+import { TEST_ANT_BALANCE } from '../../../../utils/constants';
+import { getAntConfirmations } from '../../../../utils/searchUtils';
+import {
+  AlertCircle,
+  AlertTriangleIcon,
+  BookmarkIcon,
+  BorderOuterIcon,
+  CircleCheck,
+  FileCodeIcon,
+  RefreshAlertIcon,
+} from '../../../icons';
+import Paginator from '../../../inputs/Paginator/Paginator';
+import CopyTextButton from '../../../inputs/buttons/CopyTextButton/CopyTextButton';
+import ManageAssetButtons from '../../../inputs/buttons/ManageAssetButtons/ManageAssetButtons';
+import Loader from '../../Loader/Loader';
+import RowItem from '../RowItem/RowItem';
+import './styles.css';
+
+function AntTable() {
+  const [{ arweave }] = useGlobalState();
+  const [pageRange, setPageRange] = useState<Array<number>>([0, 10]);
+  const [maxItemCount, setMaxItemCount] = useState(10);
+  const [tableItems, setTableItems] = useState([<></>]);
+
+  const [loading, setLoading] = useState(false);
+  const isMobile = useIsMobile();
+
+  useEffect(() => {
+    setLoading(true);
+    setTableItems([<></>]);
+    updateTableItems().then((items) => {
+      setTableItems(items);
+      setLoading(false);
+    });
```

Comment:
```suggestion
    }).finally(() => {
        setLoading(false);
     })
```
and remove from above

Reply:
updated